angular-todo/src/app/app.component.ts

```diff
@@ -68,7 +68,7 @@ export class AppComponent implements OnInit {
 
     Config.set({
       clientId: environment.WEB_OAUTH_CLIENT,
-      redirectUri: `${window.location.origin}/callback.html`,
+      redirectUri: `${window.location.origin}/${environment.CENTRALIZED_LOGIN === 'true' ? 'login' : 'callback.html'}`,
```

Comment:
Hmm, I think we can leave the redirectUri as the callback.html, right?

Reply:
If you leave it as 'callback.html', after successfully logging in the OAuth 2.0 flow redirects to 'OK'. 

<img width="1097" alt="image" src="https://github.com/ryanbas21/forgerock-sdk-samples/assets/156847686/95be3cf7-f3c3-4a2b-a177-687ffcb36372">

Reply:
Juan is right here. The `callback.html` is only good for background requests to `/authorize`. It's not good for the full redirects as the user gets redirected to that HTML file with no action to take.

Reply:
If it is centralized login, after successfully logging in i need to be redirected to '/login', which is why i added the conditional ternary operator.